omnibus/resources/chef/ips/symlinks.erb

```diff
@@ -0,0 +1,6 @@
+link path=usr/bin/chef-solo target=<%= projectdir %>/bin/chef-solo
```

Comment:
It seems odd this is `usr/bin/chef-solo` vs `/usr/bin/chef-solo`...is that intentional?